commands/src/main/java/org/wildfly/extras/creaper/commands/infinispan/cache/RemoveCache.java

```diff
@@ -4,11 +4,17 @@
 import org.wildfly.extras.creaper.core.online.OnlineCommandContext;
 import org.wildfly.extras.creaper.core.online.operations.Address;
 import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.Values;
 
 public final class RemoveCache implements OnlineCommand {
     private final Address address;
+    private final boolean allowResourceServiceRestart;
 
     public RemoveCache(String cacheContainer, CacheType cacheType, String cacheName) {
+        this(cacheContainer, cacheType, cacheName, false);
+    }
+
+    public RemoveCache(String cacheContainer, CacheType cacheType, String cacheName, boolean allowResourceServiceRestart) {
```

Comment:
Yea I agree, I always try to use upstream names and versions, unless there's something specific to EAP. So this is only WildFly 11+, I think? Which should be `ServerVersion.VERSION_5_0_0`.